ultralytics/engine/exporter.py

```diff
@@ -1068,6 +1099,135 @@ def export_tfjs(self, prefix=colorstr("TensorFlow.js:")):
         yaml_save(Path(f) / "metadata.yaml", self.metadata)  # add metadata.yaml
         return f, None
 
+    @try_export
+    def export_imx(self, prefix=colorstr("IMX:")):
+        """YOLO IMX export."""
+        gptq = False
+        assert LINUX, "export only supported on Linux. See https://developer.aitrios.sony-semicon.com/en/raspberrypi-ai-camera/documentation/imx500-converter"
+        if getattr(self.model, "end2end", False):
+            raise ValueError("IMX export is not supported for end2end models.")
+        if "C2f" not in self.model.__str__():
+            raise ValueError("IMX export is only supported for YOLOv8 detection models")
+        check_requirements(("model-compression-toolkit==2.1.1", "sony-custom-layers==0.2.0", "tensorflow==2.12.0"))
+        check_requirements("imx500-converter[pt]==3.14.3")  # Separate requirements for imx500-converter
+
+        import subprocess
```

Comment:
@ambitious-octopus this import is redundant

<img width="485" alt="Screenshot 2024-11-11 at 00 08 16" src="https://github.com/user-attachments/assets/cf563262-b065-43b1-a935-7e7a883fa688">